website/docs/docs/deploy/continuous-deployment.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Continuous deployment in dbt Cloud"
+sidebar_label: "Continuous deployment"
+description: "Learn about continuous deployment (CD) workflows "
+---
+
+To help you improve data transformations and ship data products faster, you can run [merge jobs](/docs/deploy/merge-jobs) to implement a continuous deployment (CD) workflow in dbt Cloud. Merge jobs can automatically build modified models whenever a pull request (PR) merges, making sure the latest code changes are in production. You don't have to wait for the next scheduled job to run to get the latest updates. 
+
+You can also implement continuous integration (CI) in dbt Cloud, which can help further to reduce the time it takes to push changes to production and improve code quality. To learn more, refer to [Continuous integration in dbt Cloud](/docs/deploy/continuous-integration). 
+
+## How merge jobs work
+
+When you set up merge jobs, dbt Cloud listens for notifications from your [Git provider](/docs/cloud/git/git-configuration-in-dbt-cloud) indicating that a PR has been merged. When dbt Cloud receives one of these notifications, it enqueues a new run of the merge job.
+
+You can set up merge jobs to perform one of the following when a PR merges:
+
+| <div style={{width:'350px'}}>Command to run</div> | Usage description |
```

Comment:
Do we anticipate that this table will grow? If not, this might read easier as a couple of bullet points

Reply:
hmm, not sure. gonna leave as is.